.github/workflows/test.yml

```diff
@@ -31,6 +31,16 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v3
 
+      # ClangFormat
+
+      - name: Install ClangFormat (Linux)
+        if: startsWith(matrix.os, 'ubuntu')
+        run: sudo apt-get install -y clang-format
+
+      - name: Install ClangFormat (macOS)
+        if: startsWith(matrix.os, 'macos')
+        run: brew install clang-format
```

Comment:
Windows runner has `LLVM` installed and it contains `clang-format` command.

https://github.com/actions/runner-images/blob/518f5407fa7ceab1db937c0b99187c88f86a754f/images/win/Windows2022-Readme.md#installed-software